moped-editor/src/views/projects/projectView/SignalComponentAutocomplete.js

```diff
@@ -1,69 +1,62 @@
 import React from "react";
-import { useState } from "react";
-import { CircularProgress } from "@material-ui/core";
+import { CircularProgress, TextField } from "@material-ui/core";
 import { Autocomplete, Alert } from "@material-ui/lab";
 import { useSocrataGeojson } from "src/utils/socrataHelpers";
 import {
-  useSignalChangeEffect,
   getSignalOptionLabel,
   getSignalOptionSelected,
-  useInitialSignalComponentValue,
-  renderSignalInput,
   SOCRATA_ENDPOINT,
 } from "src/utils/signalComponentHelpers";
 import { filterOptions } from "src/utils/autocompleteHelpers";
 
 /**
  * Material Autocomplete wrapper that enables selecting a traffic/phb signal record from a
  * Socrata dataset and setting it as a project component.
- * * @param {Object} setSelectedComponentSubtype - sets the selectedComponentSubtype from parent state
- * * @param {Object} setEditFeatureCollection - sets the editFeatureCollection from parent state
- * * @param {Boolean} editFeatureCollection - the geoJSON generated for all the the features in this component
- * * @param {Object} classes - MaterialUI style object
- *  @return {JSX.Element}
+ * @param {Object} classes - MaterialUI style object
+ * @param {Function} onChange - callback function to run when the signal is changed for React Hook Form
+ * @param {Object} value - the signal feature to set as the value of the autocomplete from React Hook Form
+ * @return {JSX.Element}
  */
-const SignalComponentAutocomplete = ({
-  classes,
-  setSelectedComponentSubtype,
-  setEditFeatureCollection,
-  editFeatureCollection,
-}) => {
-  const [signal, setSignal] = useState(null);
-  const { features, loading, error } = useSocrataGeojson(SOCRATA_ENDPOINT);
+const SignalComponentAutocomplete = React.forwardRef(
+  ({ classes, autocompleteProps, onChange, value }, ref) => {
```

Comment:
i happened to be eyeballing this in my editor and noticed you're not using `autocompleteProps` so it can be removed 🔍

Reply:
Just removed this 🙏 🕵️